ui/app/AppLayouts/Chat/controls/community/MembersDropdown.qml

```diff
@@ -0,0 +1,237 @@
+
+import shared.controls.delegates 1.0
+
+StatusDropdown {
+    id: root
+
+    property var selectedKeys: []
+    property int maximumListHeight: 288
+
+    property alias model: listView.model
+    readonly property alias count: listView.count
+
+    readonly property alias searchText: filterInput.text
+
+    signal backButtonClicked
+    signal addButtonClicked
+
+    width: 295
+
+    height: Math.min(
+                d.availableExternalHeight,
+                content.requestedHeight + d.vPadding)
+
+    padding: 11
+    bottomInset: 10
+    bottomPadding: padding + bottomInset
+
+    QtObject {
+        id: d
+
+        readonly property int sectionDelegateHeight: 40
+        readonly property int delegateHeight: 47
+
+        readonly property int vPadding: root.topPadding + root.bottomPadding
+
+        readonly property int availableExternalHeight:
+            (root.Overlay.overlay ? root.Overlay.overlay.height : 0) - root.bottomMargin -
+            (root.fixedYPosition ? contentItem.parent.y : root.topMargin)
+    }
+
+    property bool fixedYPosition: !anchors.centerIn && margins < 0
+
+    contentItem: ColumnLayout {
+        id: content
+
+        spacing: 8
+        height: root.availableHeight
+        clip: true
+
+        readonly property int requestedHeight:
+            backButton.height +
+            spacing + filterInput.height +
+            spacing + (listView.count
+                       ? Math.min(listView.contentHeight, root.maximumListHeight)
+                       : noContactsText.Layout.preferredHeight) +
+            spacing + addButton.height
+
+        StatusIconTextButton {
+            id: backButton
+
+            Layout.preferredHeight: 48//d.backButtonHeight
+            Layout.maximumWidth: root.availableWidth
+
+            spacing: 0
+            leftPadding: 4
+            statusIcon: "next"
+            icon.width: 12
+            icon.height: 12
+            iconRotation: 180
+            text: qsTr("Back")
+
+            onClicked: root.backButtonClicked()
+        }
+
+        StatusInput {
+            id: filterInput
+
+            Layout.fillWidth: true
+
+            placeholderText: qsTr("Search members")
+            maximumHeight: 36
+            topPadding: 0
+            bottomPadding: 0
+            input.asset.name: "search"
+            input.clearable: true
+
+            input.asset.width: 15
+            input.asset.height: 15
+            input.leftPadding: 13
+            input.font.pixelSize: 13
+            input.placeholder.font.pixelSize: 13
+        }
+
+        StatusBaseText {
+            id: noContactsText
+
+            Layout.fillWidth: true
+            Layout.preferredHeight: 50
+            horizontalAlignment: Text.AlignHCenter
+            verticalAlignment: Text.AlignVCenter
+
+            visible: listView.count === 0
+
+            text: qsTr("No contacts found")
+            color: Theme.palette.baseColor1
+            font.pixelSize: 12
```

Comment:
Would it make sense here to use `Theme.XXTextFontSize` instead?

Reply:
Sure!